bitnami/kafka/values-production.yaml

```diff
@@ -0,0 +1,329 @@
+logsDirs: /opt/bitnami/kafka/data
+
+## The number of threads doing disk I/O.
+numIoThreads: 8
+
+## The number of threads handling network requests.
+numNetworkThreads: 3
+
+## The default number of log partitions per topic.
+numPartitions: 1
+
+## The number of threads per data directory to be used for log recovery at startup and flushing at shutdown.
+numRecoveryThreadsPerDataDir: 1
+
+## The receive buffer (SO_RCVBUF) used by the socket server.
+socketReceiveBufferBytes: 102400
+
+## The maximum size of a request that the socket server will accept (protection against OOM).
+socketRequestMaxBytes: _104857600
+
+## The send buffer (SO_SNDBUF) used by the socket server.
+socketSendBufferBytes: 102400
+
+## Timeout in ms for connecting to zookeeper.
+zookeeperConnectionTimeoutMs: 6000
+
+## Authentication parameteres
+## https://github.com/bitnami/bitnami-docker-kafka#security
+##
+auth:
+  ## Switch to enable the kafka authentication.
+  enabled: true
+
+  ## Name of the existing secret containing credentials for brokerUser, interBrokerUser and zookeeperUser.
+  #existingSecret:
+
+  ## Name of the existing secret containing the certificate files that will be used by Kafka.
+  #certificatesSecret:
+
+  ## Password for the above certificates if they are password protected.
+  #certificatesPassword:
+
+  ## Kafka client user.
+  brokerUser: user
+
+  ## Kafka client password.
+  # brokerPassword:
+
+  ## Kafka inter broker communication user.
+  interBrokerUser: admin
+
+  ## Kafka inter broker communication password.
+  # interBrokerPassword:
+
+  ## Kafka Zookeeper user.
+  #zookeeperUser:
+
+  ## Kafka Zookeeper password.
+  #zookeeperPassword:
+
+## Kubernetes Security Context
+## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+##
+securityContext:
+  enabled: true
+  fsGroup: 1001
+  runAsUser: 1001
+
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer
+##
+service:
+  type: ClusterIP
+  port: 9092
+
+## Kafka data Persistent Volume Storage Class
+## If defined, storageClassName: <storageClass>
+## If set to "-", storageClassName: "", which disables dynamic provisioning
+## If undefined (the default) or set to null, no storageClassName spec is
+##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+##   GKE, AWS & OpenStack)
+##
+persistence:
+  enabled: true
+  # storageClass: "-"
+  accessModes:
+    - ReadWriteOnce
+  size: 8Gi
+  annotations: {}
+
+## Node labels and tolerations for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
+nodeSelector: {}
+tolerations: []
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
+  requests:
+    memory: 256Mi
```

Comment:
Given the JAVA memory requests you are setting above, it may not make sense to set 256Mi as the minimum memory request

Reply:
actually, the resource requests and limits should be commented by default.